### ISSUE TITLE
Doesn't work well in IE

### ISSUE THREAD

**Issue by EmilMoe** [ecrmnn/collect.js#124]
I get a `const is not initialized`. It points to this function, but I guess it's valid for all. Would it be possible to make it available for IE too?

~~~~
Collection.prototype.only = function (properties) {
  const collection = {};

  for (const prop in this.items) {
    if (properties.indexOf(prop) !== -1) {
      collection[prop] = this.items[prop];
    }
  }

  return new Collection(collection);
};
~~~~

**Comment by tlgreg** [comment]
The issue is the `const` in the `for ... in` iteration  scope, MS only started supporting it in Edge 14.
See compatibility table: http://kangax.github.io/compat-table/es6/

The question is, this is the only issue with IE11, Edge (<14)?

**Comment by ecrmnn** [comment]
This shouldn't be an issue now that we're transpiling to ES5.

Could you try reinstalling the module?

**Comment by ecrmnn** [comment]
Closing due to inactivity

**Comment by EmilMoe** [comment]
I'm sorry I'm on vacation in Brazil, so I can't answer so frequently.

I have the issue in IE11, is there anything I need to do or when did you change something? Because my install was from before you started transpiling to ES5?

**Comment by ecrmnn** [comment]
Did you try to reinstall the package? It should work when you get the recent version

**Comment by EmilMoe** [comment]
@ecrmnn thanks. I will try as soon as possible. Now I'm unfortunately on vacation in Brazil and only have my Macbook with me, that limits my testing abilities with IE unfortunately and the client will promptly complain if I put up a live version that is broken. So I will return back when I can.

**Comment by lauhakari** [comment]
Also getting error in IE11. Seems like not everything is being compiled down.
The error I get is on "Symbol". --> _'Symbol' is undefined_

Might need babel-polyfill I've learned :)

**Comment by lauhakari** [comment]
I tried adding babel-polyfill and then the `require("babel-polyfill");` at the top of index.js and all works fine and dandy. Want me to do a PR for it?

**Comment by ecrmnn** [comment]
@lauhakari which version are you using?

**Comment by lauhakari** [comment]
@ecrmnn 3.0.47 according to package.json, just upgraded today as I saw in the comments above it was fixed.

**Comment by lauhakari** [comment]
Clarifying my comment. I upgraded to 3.0.47 but still had the error.
_Babel Polyfill_ did fix this error for me, with the `_Symbol_ is undefined`

**Comment by lauhakari** [comment]
Any word?
The app we are usibg collect.js in isn't working in <IE11 because of the SYMBOL problem.

**Comment by ecrmnn** [comment]
@lauhakari  @EmilMoe 

Should be fixed in 3.0.49.

For future reference please create new issues instead of commenting on closed ones. I do not watch new comments in closed issues.
